### ISSUE TITLE
Error on install

### ISSUE THREAD

**Issue by fufroma** [mduplouy/acme-web-manager#1]
Hello,

When launching composer.phar install, there is an warning with a dependency

```
  - Installing octopuce/acmephpc (dev-pimple-3.x 6460259)
    Cloning 6460259dac464e8e67fa34ca5eace0badd9f5bcc from cache
    6460259dac464e8e67fa34ca5eace0badd9f5bcc is gone (history was rewritten?)
    Failed to download octopuce/acmephpc from source: Failed to execute git checkout '6460259dac464e8e67fa34ca5eace0badd9f5bcc' -- && git reset --hard '6460259dac464e8e67fa34ca5eace0badd9f5bcc' --

fatal: reference is not a tree: 6460259dac464e8e67fa34ca5eace0badd9f5bcc

    Now trying to download from dist
  - Installing octopuce/acmephpc (dev-pimple-3.x 6460259)
    Downloading: 100%     
```

**Comment by mduplouy** [comment]
Should be fixed by 9dcaed8973fdb75dd5488412b7f06fc57231964e

**Comment by fufroma** [comment]
Fixed, thx
